### PR TITLE
Add a 1s timeout before fetching equipments infos 

### PR DESCRIPTION
**Pull request by sBouzols**
between searchMatchingEquipments callings (between keyboard inputs)

Signed-off-by: sBouzols <sylvain.bouzols@gmail.com>

### CODE REVIEW

File: src/components/dialogs/equipment-search-dialog.js [hunk 1]
```diff
@@ -51,35 +51,43 @@ const EquipmentSearchDialog = ({
...
+                                getEquipmentsInfosForSearchBar(
+                                    infos,
+                                    useNameLocal
+                                )
+                            );
+                        } // else ignore results of outdated fetch
                     })
-                );
+                    .catch((errorMessage) =>
+                        displayErrorMessageWithSnackbar({
```
Comment (SlimaneAmar) [inline src/components/dialogs/equipment-search-dialog.js:81]: use useSnackMessage ?

File: src/components/app-top-bar.js [hunk 2]
```diff
@@ -167,33 +167,41 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
...
+                        displayErrorMessageWithSnackbar({
+                            errorMessage: errorMessage,
+                            enqueueSnackbar: enqueueSnackbar,
+                            headerMessage: {
+                                headerMessageId: 'equipmentsSearchingError',
+                                intlRef: intlRef,
+                            },
+                        })
+                    );
+            }, 1500);
```
Comment (SlimaneAmar) [inline src/components/app-top-bar.js:204]: Why 1,5 s ?
Create a parameter ?

File: src/components/app-top-bar.js [hunk 3]
```diff
@@ -167,33 +167,41 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
...
+                                getEquipmentsInfosForSearchBar(
+                                    infos,
+                                    useNameLocal
+                                )
+                            );
+                        } // else ignore results of outdated fetch
                     })
-                );
+                    .catch((errorMessage) =>
+                        displayErrorMessageWithSnackbar({
```
[the remark this answers is not in the document]
Comment (sBouzols) [reply src/components/app-top-bar.js]: done

File: src/components/dialogs/equipment-search-dialog.js (hunk 1 above)
Comment (sBouzols) [reply src/components/dialogs/equipment-search-dialog.js]: done

**Review by SlimaneAmar — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]
Ok with 1 sec delay

**Pull request merged**